build-system/tasks/helpers.js

```diff
@@ -294,11 +313,31 @@ function compileMinifiedJs(srcDir, srcFilename, destDir, options) {
         }
       })
       .then(() => {
-        let name = options.minifiedName;
+        let name = minifiedName;
         if (options.latestName) {
           name = `${name} → ${options.latestName}`;
         }
+        if (options.singlePassCompilation) {
+          altMainBundles.forEach(bundle => {
+            name = `${name}, ${bundle.name}.js`;
+          });
+          name = `${name}, and all extensions`;
+        }
         endBuildStep('Minified', name, startTime);
+      })
+      .then(() => {
+        if (argv.fortesting && MINIFIED_TARGETS.includes(minifiedName)) {
+          return enableLocalTesting(`${destDir}/${minifiedName}`);
+        }
+      })
+      .then(() => {
+        if (options.singlePassCompilation) {
+          const promises = [];
+          altMainBundles.forEach(bundle => {
+            promises.push(enableLocalTesting(`dist/${bundle.name}.js`));
```

Comment:
Q: what I understand here is that for now all single-pass compilations are enabled for testing regardless of whether `--fortesting` is passed? Will that change in the future?

Reply:
Good catch! This too should have been gated on `--fortesting`. Fixed.